### PR TITLE
Update Fetch Events Query name and description

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/37865

## Description
Update Fetch Events Query name and description

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@guykeller @guyfreund Please review this, and approve if make sense.

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml [hunk 1]
```diff
@@ -20,13 +20,12 @@ configuration:
...
-  display: Fetch notable events ES query
+  display: Fetch events query
   name: fetchQuery
   required: false
   type: 0
-  additionalinfo: The query for the notable events enrichment (defined by default).
-    If you decide to edit this, make sure to provide a query that uses the \`notable\`
-    macro. Use the new default query as an example.
+  additionalinfo: You can edit this query to fetch other types of events. 
+    Note that to fetch notable events, make sure the query uses the \`notable\` macro.
```
Comment (guykeller) [inline Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml:28]: This line is unnecessary in my opinion, they are aware of it and we're not going to teach them SPL in my opinion

Comment (guykeller) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml:28]: Alternatively, I would write something like: "The Splunk search query to fetch events by", every Splunk user would understand

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml (hunk 1 above)
Comment (guyfreund) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml:28]: We need to make sure that new customers are using the notable macro to build their environment (preprocess, pbs, etc...) with the notable macro. So I think we should keep this.
```suggestion
    Note that to fetch ES notable events, make sure to include the \`notable\` macro in your query.
```

**Comment by ShirleyDenkberg** [comment]
Moved this change to the Splunk doc changes branch so all changes are in a single branch.  See https://github.com/demisto/content/pull/13216

**Pull request closed without merging**